src/browser/components/MainPage.jsx

```diff
@@ -394,14 +404,13 @@ export default class MainPage extends React.Component {
   }
 
   render() {
-    const self = this;
     let tabsRow;
-    if (this.props.teams.length > 1) {
+    if (this.state.teams.length > 1) {
```

Comment:
`Config` listener callback will update state on `update`, thus making it more appropriate to drive component by state instead of props.